docs/modules/ROOT/pages/how-tos/cloudscale/replace-storage-node.adoc

```diff
@@ -0,0 +1,77 @@
+= Replace a storage node
+
+:cloud_provider: cloudscale
+:kubectl_extra_args: --as=cluster-admin
+:delabel_app_nodes: yes
+:argo_app: rook-ceph
+
+:osd-replace-list: $NODE_TO_REPLACE
+
+:mon-operation: replace
+:mon-argocd-autosync-already-disabled: yes
+:mon-replace-list: $NODE_TO_REPLACE
+
+:node-delete-list: ${NODE_TO_REPLACE}
+:delete-nodes-manually: yes
+:delete-pvs: old_pv_names
```

Comment:
Again very minor, but we don't delete pvs, so I don't think we need to set this to anything